app/assets/stylesheets/application.css

```diff
@@ -45,3 +45,73 @@
 #footer-logo-zense{
   width: 75%;
 }
+
+
+/*login/signup css starts*/
+.middle-block {
+    display: table;
+    margin:auto;
+  }
+
+   .center-block {
+    display: block;
+    margin-right: auto;
+    margin-left: auto;
+   }
+
+  .custom-submit {
+    color: #FFFFFF;
+    height: 51px;
+    font-size: 22px !important;
```

Comment:
It's generally a good idea to not use `!important` 
Reference - https://uxengineer.com/css-specificity-avoid-important-css/

Reply:
updated